internal/ramses/secure.go

```diff
@@ -0,0 +1,21 @@
+package ramses
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+// Secure header
+type Secure struct {
+	IPAddress      uint32
+	Port           uint16
+	Seq            uint16
+	Retransmission uint16
+	Ack            uint16
+	_              uint32
+}
+
+// Read fills a structure with data
+func (secure *Secure) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.LittleEndian, secure)
```

Comment:
how about a test of a real Secure buffer from a `.rac` file resulting in the expected struct?